urllib3/util/sessioncontext.py

```diff
@@ -0,0 +1,75 @@
+import time
+
+from ..packages.six.moves.http_cookiejar import (
+    DefaultCookiePolicy as PythonCookiePolicy,
+    CookieJar as PythonCookieJar
+)
+
+
+class DefaultCookiePolicy(PythonCookiePolicy):
+    """
+    The default urllib3 cookie policy - similar to the Python default,
+    but :param:`strict_ns_domain` is set to `DomainStrict` for security.
+    """
+    def __init__(self, *args, **kwargs):
+        policy = PythonCookiePolicy.DomainStrict
+        kwargs.setdefault('strict_ns_domain', policy)
+        # Old-style class on Python 2
+        PythonCookiePolicy.__init__(self, *args, **kwargs)
+
+
+class CookieJar(PythonCookieJar):
+
+    def __init__(self, policy=None):
+        policy = policy or DefaultCookiePolicy()
```

Comment:
We maybe want to use an explicit `is not None` check here, rather than rely on the truthiness of arbitrary objects.